libs/wb-types/src/weapon.ts

```diff
@@ -42,17 +42,24 @@ export enum Weapon {
     SCAR = "p93",
     TacticalShotgun = "p94",
     VEK = "p95",
-    UNKNOWN_WEAPON_96 = "p96",
-    UNKNOWN_WEAPON_97 = "p97",
+    Desert = "p96",
+    Auto = "p97",
     LMG = "p98",
-    UNKNOWN_WEAPON_103 = "p103",
-    UNKNOWN_WEAPON_104 = "p104",
-    UNKNOWN_WEAPON_105 = "p105",
-    UNKNOWN_WEAPON_101 = "p101",
-    UNKNOWN_WEAPON_110 = "p110",
+    UNRELEASED = "p99",
```

Comment:
replace `UNRELEASED` -> `UNRELEASED_WEAPON_99` for disambiguation with other unreleased weapons